### DISCUSSION TITLE
TTS和whisper希望可以用国内的方案，比如腾讯的，阿里的

### DISCUSSION THREAD

**Discussion by chengqiyi** [FoloToy/folotoy-server-self-hosting#93] [General | 讨论]
经过测试，LLM模型可以使用API2D等代理网站，但是对TTS和whisper的支持都不太好
而且TTS和whisper，国内方案的水平也不错，用在中文上效果也更好。
如果能支持国内的方案，会大大降低使用门槛，届时，用户完全不需要有一个openai账号才能玩起来

**Comment by marvyn** [comment]
国内的 TTS，STT，会有很多审查、敏感词过滤吧。
国外的 LLM 模型返回的结果，国内 TTS 不返回结果，是很可能的

**Comment by lewangdev** [comment]
国内 stt 和 tts 还有哪些方案？测试过百度的 stt ，效果太差了

**Comment by chengqiyi** [comment]
试试腾讯呢，腾讯的语音转文字我们公司用过还可以。
关于内容审核，好像openai是没有对内容进行审核的，Azure 也是会对内容有审核。不过这些也可以通过prompt来进行一些限制



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: marvyn ***@***.***&gt;
发送时间: 2023年11月23日 23:25
收件人: FoloToy/folotoy-server-self-hosting ***@***.***&gt;
抄送: bigcoco ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [FoloToy/folotoy-server-self-hosting] TTS和whisper希望可以用国内的方案，比如腾讯的，阿里的 (Issue #73)





 
国内的 TTS，STT，会有很多审查、敏感词过滤吧。
 国外的 LLM 模型返回的结果，国内 TTS 不返回结果，是很可能的
